wcomponents-test-lib/src/test/java/com/github/bordertech/wcomponents/test/components/WTabSetUI.java

```diff
@@ -0,0 +1,24 @@
+package com.github.bordertech.wcomponents.test.components;
+
+import com.github.bordertech.wcomponents.WComponent;
+import com.github.bordertech.wcomponents.WPanel;
+import com.github.bordertech.wcomponents.WTabSet;
+import com.github.bordertech.wcomponents.WText;
+
+public class WTabSetUI extends WPanel {
+
+	private WPanel panel = new WPanel();
+
+	public WTabSetUI() {
+		WTabSet tabSet = new WTabSet();
+		tabSet.addTab(createDummyPanelWithText("One"), "First Tab");
+		tabSet.addTab(createDummyPanelWithText("Two"), "Second Tab");
+		panel.add(tabSet);
+		add(panel);
+	}
+
+	private WComponent createDummyPanelWithText(String name) {
+		return new WText("Tab: " + name);
+	}
+
+}
```

Comment:
same as previous - expose the WTabSet to allow for testing of all states and properties. It may also be worthwhile exposing the tabs to test their states and properties too.